server/tests/src/test/java/org/infinispan/server/resilience/ResilienceMaxRetryIT.java

```diff
@@ -76,7 +74,7 @@ public void testMaxRetries0() {
          }
       }
 
-      // 6 -> kill port2 and port3, start port1
+      // 6 -> kill server1 and server2, start server0
```

Comment:
Adding the comment here because lines 83-83 haven't changed:

Separating steps 7 and 8 seems very artificial, and the per-step comments don't say anything useful.
This is a repeat of step 5, only with the a topology reset instead of a topology update:

```
7 -> wait for the client to resets the topology to the initial server list
The client resets to the initial server list when all servers are marked as failed,
During each cache operation, the connection fails and the target server is marked as failed,
but we don't control the target server so we don't know how many operations we need.
When an operation succeeds, we know the client has reset its topology
and has connected to server0.
```